packages/content-fetch/src/request_handler.ts

```diff
@@ -147,11 +148,29 @@ export const contentFetchRequestHandler: RequestHandler = async (req, res) => {
       logRecord.error = 'unknown error'
     }
 
+    // capture error event
+    users.forEach((user) => {
+      analytics.capture({
+        distinctId: user.id,
+        event: 'content-fetch-failure',
+        properties: logRecord,
```

Comment:
I think `logRecord` might have too much stuff in it. Maybe just send the URL?

Reply:
Yeah, good catch.